jax/_src/scipy/signal.py

```diff
@@ -33,6 +33,68 @@
 from jax._src.typing import Array, ArrayLike
 from jax._src.util import canonicalize_axis, tuple_delete, tuple_insert
 
+@_wraps(osp_signal.fftconvolve)
+def fftconvolve(in1: Array ,in2: Array, mode: str ="same",axes=None) -> Array:
+  # sanity checks
+  in1 = jnp.asarray(in1)
+  in2 = jnp.asarray(in2)
+  if in1.ndim == in2.ndim == 0:  # scalar inputs
+    return in1 * in2
+  elif in1.ndim != in2.ndim:
+      raise ValueError("in1 and in2 should have the same dimensionality")
+  elif in1.size == 0 or in2.size == 0:  # empty arrays
+        return jnp.array([])
+  # warn current limitations
+  if mode != "same":
+    warnings.warn("current fftconvolve() use only mode=same argument")
+    mode = "same"
+  if axes is not None:
+    warnings.warn("fftconvolve() do not use axes argument")
+
+  s1 = in1.shape
+  s2 = in2.shape
+
+  axes = range(in1.ndim)
+  # Axes of length 1 can rely on broadcasting rules for multipy,
+  # no fft needed.
+  axes = [a for a in axes if s1[a] != 1 and s2[a] != 1]
+
+  if not all(s1[a] == s2[a] or s1[a] == 1 or s2[a] == 1
+             for a in range(in1.ndim) if a not in axes):
+    raise ValueError("incompatible shapes for in1 and in2:"
```

Comment:
You may be able to use existing utilities here, e.g. https://github.com/google/jax/blob/6f2256ad17304ab2be667cdfc49302125220b81c/jax/_src/scipy/signal.py#L281-L286